tools/dkcm/clerk/clerk.go

```diff
@@ -0,0 +1,94 @@
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package clerk
+
+import (
+	"database/sql"
+	"math/rand"
+	"time"
+	// _ "github.com/lib/pq"
+)
+
+type Cluster struct {
+	clusterid   string
+	accesstoken string
+	boskosid    string
+	prowid      string
+	status      string
+	zone        string
+}
+
+const charset = "abcdefghijklmnopqrstuvwxyz" +
+	"ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"
+
+var (
+	db        *sql.DB
+	randomGen *rand.Rand = rand.New(
+		rand.NewSource(time.Now().UnixNano()))
+)
+
+/*
+func init() {
```

Comment:
Please remove the code that are commented out. Ditto for the other places in this PR.